recipes/glew/run_test.bat

```diff
@@ -0,0 +1,7 @@
+conda env create -f %RECIPE_DIR%\test\test-environment-windows.yml
```

Comment:
The real solution is to get issue ( https://github.com/conda/conda-build/issues/1059 ) resolved.